### PR TITLE
Cleaning up some data

### PR DESCRIPTION
**Pull request by minh-v**
Did some initial scrolling through dataset to fix errors and make some changes I thought would be helpful.
Any feedback would be appreciated!

### CODE REVIEW

File: alpaca_data_cleaned.json [hunk 1]
```diff
@@ -667,7 +667,7 @@
     {
         "instruction": "Classify the following data with three labels.",
         "input": "fjsklfjdsklfjsklfjsklfjs",
-        "output": "Label1: Character data\nLabel2: Alphabetical data\nLabel3: Non-alphabetical data"
+        "output": "1. Alphabetical Data\n2. Numeric Data\n3. Alpha-numeric data"
```
Comment (gururise) [inline alpaca_data_cleaned.json:670]: Should be:
instruction: "Classify the following data with one of three labels:\n1. Alphabetical Data\n2. Numeric Data\n3. Alpha-numeric data"
output: "1. Alphabetical Data"

File: alpaca_data_cleaned.json [hunk 2]
```diff
@@ -517,10 +517,10 @@
     {
         "instruction": "Generate a password of 8 characters.",
         "input": "",
-        "output": "e9gKyd2D"
+        "output": "e^gK!2Dg"
```
Comment (gururise) [inline alpaca_data_cleaned.json:520]: Unnecessary change. Answer was already technically correct.

File: alpaca_data_cleaned.json [hunk 3]
```diff
@@ -73287,7 +73287,7 @@
     {
         "instruction": "Create a math equation where the answer is 3.",
         "input": "",
-        "output": "1 + 1 + 1 = 3"
+        "output": "1 + 2 = 3"
```
Comment (gururise) [inline alpaca_data_cleaned.json]: Unnecessary.  Answer was already correct.

File: alpaca_data_cleaned.json [hunk 4]
```diff
@@ -74372,7 +74372,7 @@
     {
         "instruction": "Create an encryption key and decipher this message.",
         "input": "Kjgk yhjy yudt trgf",
-        "output": "Encryption Key: ABCDEFGHIJKLMNOPQRSTUVWXYZ\nDeciphered Message: Hello World!"
+        "output": "Encryption key: Let's use a simple substitution cipher, where each letter of the alphabet is shifted forward by 3 places.\nDeciphered message: Hello there hope you are"
```
Comment (gururise) [inline alpaca_data_cleaned.json]: The input should match the deciphered message after applying the encryption key.

File: alpaca_data_cleaned.json [hunk 5]
```diff
@@ -74762,7 +74762,7 @@
     {
         "instruction": "Construct a true sentence with the words \"rat\", \"hat\", and \"mat\".",
         "input": "",
-        "output": "The rat wore a hat while sitting on the mat."
+        "output": "The rat sat on the mat while wearing a hat."
```
Comment (gururise) [inline alpaca_data_cleaned.json]: Unnecessary change. Output was already correct.

File: alpaca_data_cleaned.json [hunk 6]
```diff
@@ -74902,7 +74902,7 @@
     {
         "instruction": "Convert 1 kilometer to meters.",
         "input": "1 kilometer",
-        "output": "1 kilometer is equal to 1000 meters."
+        "output": "One kilometer is equal to 1000 meters. Therefore 1 kilometer is equal to 1000 meters."
```
Comment (gururise) [inline alpaca_data_cleaned.json:74905]: `"output": "One kilometer is equal to 1000 meters. Therefore the answer is 1000 meters."`

File: alpaca_data_cleaned.json [hunk 7]
```diff
@@ -75572,7 +75572,7 @@
     {
         "instruction": "Generate a headline for a blog post about machine learning.",
         "input": "",
-        "output": "Unlocking the Power of Machine Learning for the Next Generation of Businesses."
+        "output": "Revolutionizing the Future: How Machine Learning is Changing the World We Live In"
```
Comment (gururise) [inline alpaca_data_cleaned.json]: Unnecessary Change. Output was already correct.

File: alpaca_data_cleaned.json [hunk 8]
```diff
@@ -75592,7 +75592,7 @@
     {
         "instruction": "Generate a metaphor for a driverless car",
         "input": "",
-        "output": "A driverless car is like a ship without a captain navigating an ocean of traffic."
+        "output": "A driverless car is like a ghost on wheels, moving through the streets without a visible driver, guided only by an unseen force."
```
Comment (gururise) [inline alpaca_data_cleaned.json]: Unnecessary Change. Output was already correct.

File: alpaca_data_cleaned.json [hunk 9]
```diff
@@ -75887,7 +75887,7 @@
     {
         "instruction": "Create a riddle involving animals.",
         "input": "",
-        "output": "What has four legs but can't walk? \nA table."
+        "output": "I'm an animal that's small and cute, \nI'm often kept as a pet, that's no dispute, \nI come in many colors, and I can run quite fast, \nIn a wheel or a ball, I have a blast! \nAnswer: A hamster."
```
Comment (gururise) [inline alpaca_data_cleaned.json:75890]: `"output": "I'm an animal that's small and cute,\nI'm often kept as a pet, that's no dispute,\nI come in many colors, and I can run quite fast,\nIn a wheel or a ball, I have a blast!\nWhat am I?\nAnswer: A hamster."`

File: alpaca_data_cleaned.json [hunk 10]
```diff
@@ -75940,7 +75940,7 @@
         "output": "(123) 456-7890"
     },
     {
-        "instruction": "Change the following word to its synonym.",
+        "instruction": "Give a synonym for the following word.",
```
Comment (gururise) [inline alpaca_data_cleaned.json]: Unnecessary Change. Output already correct.

File: alpaca_data_cleaned.json [hunk 11]
```diff
@@ -77077,7 +77077,7 @@
     {
         "instruction": "Write a sentence using a simile.",
         "input": "",
-        "output": "The clouds looked like giant cotton candy in the sky."
+        "output": "The wind howled through the trees like a pack of wolves on the hunt."
```
Comment (gururise) [inline alpaca_data_cleaned.json:77080]: Unnecessary Change. Output already correct.

File: alpaca_data_cleaned.json [hunk 12]
```diff
@@ -77267,15 +77267,15 @@
...
...sions, coding skills are essential for success. This can be seen in fields such as software engineering and web development. However, coding skills are also extremely valuable in many other industries, such as finance and healthcare. As such, developing coding skills in schools is essential to providing students with the skills necessary to succeed in the modern job market.\n\nFinally, coding can be an important creative outlet for many students. While coding can be a useful tool for educational or job-related purposes, it can also be a source of joy and creative exploration. Through coding, students can create digital art, develop unique websites and applications, and explore their own creativity. In this way, coding not only provides students with an academic and professional advantage, but also offers them an enjoyable and expressive outlet.\n\nIn conclusion, coding is an important skill for today's students to learn. Schools should be providing an environment where students can gain the skills necessary to navigate our digital world. By delivering instruction in coding, schools can give their students the skills and knowledge needed to adapt and thrive in the modern workplace."
```
Comment (gururise) [inline alpaca_data_cleaned.json]: What is the purpose of this change?

File: alpaca_data_cleaned.json [hunk 13]
```diff
@@ -77285,7 +77285,7 @@
         "output": "Sing to the tune of \"The Wheels on the Bus\":\n\nGo, go, go have fun,\nDo your best and you'll be done,\nBe brave and show your pride,\nLet's all go for a ride,\nGo, go, go have fun!"
     },
     {
-        "instruction": "Name at least three popular animals in the United States.",
+        "instruction": "Name at three popular animals in the United States.",
```
Comment (gururise) [inline alpaca_data_cleaned.json:77288]: What was the purpose of this change?

File: alpaca_data_cleaned.json [hunk 14]
```diff
@@ -96622,7 +96622,7 @@
...
+        "output": "Pegasus Design Studio is the premier design firm for all of your business needs. We specialize in website design, user experience design, digital marketing, corporate branding, and commercial printing to ensure you are equipped with the tools to succeed.\n\nAt Pegasus Design Studio, we work with a variety of organizations in various industries such as technology, retail, food and beverage, healthcare, and more. Our team of experienced professionals provide creative yet functional web designs, logos, business cards, prints and more.\n\nWe pride ourselves on delivering innovative and exceptional services to our clients. Our team works diligently to ensure that your business objectives are met and exceeded. Our commitment to excellence can be felt in every aspect of our work, from the initial concept to the finished project.\n\nWe understand that each business has unique needs and our team is always looking for groundbreaking solutions to meet those needs. Our clients come first and our satisfaction guarantee is unmatched.\n\nIf you are looking for top-notch design services, look no further than Pegasus Design Studio!"
```
Comment (gururise) [inline alpaca_data_cleaned.json:96625]: ?

File: alpaca_data_cleaned.json [hunk 15]
```diff
@@ -189352,13 +189352,13 @@
...
+        "output": "One example of a real-world problem that can be solved with GPT is natural language processing. Natural language processing involves understanding the structure, meaning and function of spoken sentences to form automatic responses or actions. GPT can be used to develop natural language processing models that can be used in applications like chatbots, document summarization, text-to-speech conversion, machine translation, and more. GPT technologies can also be used for text recognition and understanding for tasks such as text classification, text summarization and deep text understanding. GPT can also be used to develop natural language models that understand complex language structures, as well as extract relevant information from large bodies of text. Additionally, tasks such as information extraction, dialog systems, question answering and image caption generation can be implemented using GPT technologies.  This can help automate many tasks, allowing humans to save time and effort. By applying GPT to natural language processing, the possibilities are endless."
```
Comment (gururise) [inline alpaca_data_cleaned.json:189355]: ?

**Review by gururise — CHANGES_REQUESTED** [review]

File: alpaca_data_cleaned.json (hunk 14 above)
Comment (gururise) [reply alpaca_data_cleaned.json:96625]: Oh, i see the space. this one is fine!

Comment (minh-v) [reply alpaca_data_cleaned.json:96625]: Uniform formatting.

File: alpaca_data_cleaned.json (hunk 15 above)
Comment (minh-v) [reply alpaca_data_cleaned.json:189355]: Uniform formatting.
All other "output":  has a space in front

File: alpaca_data_cleaned.json (hunk 13 above)
Comment (minh-v) [reply alpaca_data_cleaned.json:77288]: Correct grammer

File: alpaca_data_cleaned.json (hunk 12 above)
Comment (minh-v) [reply alpaca_data_cleaned.json:77270]: Update to the current year

File: alpaca_data_cleaned.json (hunk 11 above)
Comment (minh-v) [reply alpaca_data_cleaned.json:77080]: I thought this simile was a better example.

File: alpaca_data_cleaned.json (hunk 5 above)
Comment (minh-v) [reply alpaca_data_cleaned.json]: I thought this format sounded a lot better.

File: alpaca_data_cleaned.json (hunk 4 above)
Comment (minh-v) [reply alpaca_data_cleaned.json]: What do you mean? Might just revert this one.

File: alpaca_data_cleaned.json (hunk 11 above)
Comment (gururise) [reply alpaca_data_cleaned.json:77080]: Sure, we can keep this change. Its still a simile and correct answer.

File: alpaca_data_cleaned.json (hunk 12 above)
Comment (gururise) [reply alpaca_data_cleaned.json]: \u2019 is unicode, not the year.

File: alpaca_data_cleaned.json (hunk 13 above)
Comment (gururise) [reply alpaca_data_cleaned.json:77288]: "Name at three popular animals in the United States." sounds wrong to me. Missing the word 'least'

Comment (gururise) [reply alpaca_data_cleaned.json:77288]: or get rid of 'at'?

**Review by gururise — APPROVED** [review]
Thanks for the changes!! After review, it looks great!
